client/my-sites/site-settings/section-writing.jsx

```diff
@@ -20,9 +20,9 @@ module.exports = React.createClass( {
 	render: function() {
 
 		return (
-			<Card className="writing-settings">
-				<WritingForm site={ this.props.site } />
-			</Card>
+
+			<WritingForm site={ this.props.site } />
+
 		);
 
 	}
```

Comment:
As simple as this component is, would you mind updating it to use ES6 syntax (if the component is here to stay)? Nothing too much would change except for using `import` instead of `var X = require( Y );` and `module.exports` could be changed to `export default React.createClass( … )`. We could also use the function short-hand like `render() { … }` instead of `render: function() { … }`